docs/ios/v2/advanced-features/simulcast.mdx

```diff
@@ -0,0 +1,37 @@
+---
+ title: Simulcast
+ nav: 4.01
+---
+
+## Minimum Requirements
+
+- SDK version 0.5.1 or higher
+- Simulcast enabled in room template
+
```

Comment:
I think it will be good to explain Simulcast feature a bit at the beginning and how clients could benefit from it.

Reply:
agree. I loved the intro section that you had in this doc on Notion - would be great to have a line or two here. also check out #921